pkg/reporter/format.go

```diff
@@ -0,0 +1,41 @@
+package reporter
+
+import (
+	"fmt"
+	"io"
+	"os"
+
+	"golang.org/x/term"
+)
+
+var format = []string{"table", "json", "markdown", "sarif"}
+
+func Format() []string {
+	return format
+}
+
+func GetReporter(format string, stdout, stderr io.Writer, fileOutput bool) (Reporter, error) {
```

Comment:
`fileOutput` is a bit of a weird interface. Can we have this take in a `terminalWidth` instead, and just have the caller provide it?

Reply:
Done in: 5f1fd7a471c91a6065a92814389ab6399a399bee